### PR TITLE
Fix openstack command not found issue

### PR DESCRIPTION
**Pull request by psathyan**
Signed-off-by: Pragadeeswaran Sathyanarayanan <psathyan@redhat.com>

# Description

This PR fixes the below error observed

```
+ .venv/bin/python /home/jenkins-build/workspace/rhos_quota/utility/psi_quota.py --osp-cred /home/jenkins-build/osp-cred-ci-2.yaml
/home/jenkins-build/workspace/rhos_quota/.venv/lib64/python3.6/site-packages/paramiko/transport.py:33: CryptographyDeprecationWarning: Python 3.6 is no longer supported by the Python core team. Therefore, support for it is deprecated in cryptography and will be removed in a future release.
  from cryptography.hazmat.backends import default_backend
/home/jenkins-build/workspace/rhos_quota/run.py:6: MonkeyPatchWarning: Monkey-patching ssl after ssl has already been imported may lead to errors, including RecursionError on Python 3.6. It may also silently lead to incorrect behaviour on Python 3.7. Please monkey-patch earlier. See https://github.com/gevent/gevent/issues/1016. Modules that had direct imports (NOT patched): ['urllib3.util.ssl_ (/home/jenkins-build/workspace/rhos_quota/.venv/lib64/python3.6/site-packages/urllib3/util/ssl_.py)', 'urllib3.util (/home/jenkins-build/workspace/rhos_quota/.venv/lib64/python3.6/site-packages/urllib3/util/__init__.py)']. 
  monkey.patch_all()
Traceback (most recent call last):
  File "/home/jenkins-build/workspace/rhos_quota/utility/psi_quota.py", line 277, in <module>
    run(arguments)
  File "/home/jenkins-build/workspace/rhos_quota/utility/psi_quota.py", line 235, in run
    cmd=openstack_basecmd(**os_cred) + " server list -f json"
  File "/home/jenkins-build/workspace/rhos_quota/utility/psi_quota.py", line 35, in execute
    proc = subprocess.Popen(cmdlist, stdout=stdout, stderr=stderr)
  File "/home/jenkins-build/workspace/rhos_quota/.venv/lib64/python3.6/site-packages/gevent/subprocess.py", line 822, in __init__
    start_new_session)
  File "/home/jenkins-build/workspace/rhos_quota/.venv/lib64/python3.6/site-packages/gevent/subprocess.py", line 1837, in _execute_child
    raise child_exception
FileNotFoundError: [Errno 2] No such file or directory: 'openstack'
```

There was another issue observed during the fix
```
Error 111: Connection refused
```
this occurred due to missing `postfix` that was required by `smtplib` to send e-mails in the python scripts.

__Log__
https://ceph-downstream-jenkins-csb-storage.apps.ocp-c1.prod.psi.redhat.com/view/RHCS%20QE/job/rhos_quota/276/console

### CODE REVIEW

File: pipeline/rhos_scripts/Jenkinsfile-quota.groovy [hunk 1]
```diff
@@ -25,7 +25,7 @@ node("rhel-8-medium || ceph-qe-ci") {
                         reference: ''
                     ]],
                     userRemoteConfigs: [[
-                        url: "https://github.com/red-hat-storage/cephci.git"
+                        url: "https://github.com/psathyan/cephci.git"
```
Comment (tintumathew10) [inline pipeline/rhos_scripts/Jenkinsfile-quota.groovy:28]: git repository is pointing to https://github.com/psathyan/cephci.git. is this intentional?

File: pipeline/rhos_scripts/Jenkinsfile-quota.groovy [hunk 2]
```diff
@@ -10,7 +10,7 @@ node("rhel-8-medium || ceph-qe-ci") {
             checkout(
                 scm: [
                     $class: 'GitSCM',
-                    branches: [[name: "origin/master"]],
+                    branches: [[name: "origin/fixQuota"]],
```
Comment (tintumathew10) [inline pipeline/rhos_scripts/Jenkinsfile-quota.groovy:13]: are we proceeding with fixQuota instead of master branch?

**Review by tintumathew10 — CHANGES_REQUESTED** [review]

File: pipeline/rhos_scripts/Jenkinsfile-quota.groovy (hunk 1 above)
Comment (psathyan) [reply pipeline/rhos_scripts/Jenkinsfile-quota.groovy]: Thanks... I observed one more failure during unit testing... hence made a change to test it. After the unit testing is complete, I'll revert this change.

File: pipeline/rhos_scripts/Jenkinsfile-quota.groovy (hunk 2 above)
Comment (psathyan) [reply pipeline/rhos_scripts/Jenkinsfile-quota.groovy]: This change would be reverted after unit testing is successful.

**Review by tintumathew10 — APPROVED** [review]
LGTM

**Pull request merged**